src/minidexed.cpp

```diff
@@ -567,9 +567,9 @@ void CMiniDexed::SetMIDIChannel (uint8_t uchChannel, unsigned nTG)
 		}
 	}
 
-	assert (nActiveTGs <= 8);
-	static const unsigned Log2[] = {0, 0, 1, 2, 2, 3, 3, 3, 3};
-	m_nActiveTGsLog2 = Log2[nActiveTGs];
+	//assert (nActiveTGs <= 16);
+	//static const unsigned Log2[] = {0, 0, 1, 2, 2, 3, 3, 3, 3};
+	//m_nActiveTGsLog2 = Log2[nActiveTGs];
```

Comment:
What was this doing? Can it just be commented out?

Reply:
I don't know why any of this code was in there - I can only assume it was relevant at some point and then no longer required.  It certainly isn't used at the moment anywhere I could find, hence taking it out.

Kevin